Makefile

```diff
@@ -1,6 +1,13 @@
 all: lkmpg.tex
 	pdflatex -shell-escap lkmpg.tex
 
+html: lkmpg.tex
+	make4ht -suf html5 -c html.cfg -d html lkmpg.tex
+
 clean:
 	rm -f *.dvi *.aux *.log *.ps *.pdf *.out lkmpg.bbl lkmpg.blg lkmpg.lof lkmpg.toc
-	rm -rf _minted-lkmpg
+	rm -rf _minted-lkmpg html
+# The following command is for html files
```

Comment:
The comment is not necessary since the filenames already imply. Remove this line for simplicity.

Reply:
done